script.js

```diff
@@ -1,34 +1,39 @@
+//Calculate Tip
+
 function calculateTip() {
-  var billTot = document.getElementById("billTot").value;
-  var tipPer = document.getElementById("tipPer").value;
-  var split = document.getElementById("split").value;
-
- //validate input
- if (billTot === "" || serviceQual == 0) {
-  alert("Please enter values");
-  return;
-}
+    var billAmt = document.getElementById("billAmt").value;
```

Comment:
I'm not a fan of shortened variable names. Yes they save a couple of key strokes but the clarity you get by typing out the whole word is worth it in my experience.